backends/transforms/utils.py

```diff
@@ -187,3 +230,128 @@ def delete_constant_placeholder(exp_program: ExportedProgram, node: torch.fx.Nod
 
     # Remove node from graph
     node.graph.erase_node(node)
+
+
+def create_mutable_buffer(
+    exp_program: ExportedProgram,
+    graph: torch.fx.Graph,
```

Comment:
theres a 1:1 mapping of EP to graph why bother passing as separate arg?

Reply:
fixed in the follow up.